frontend/src/layout/nav.vue

```diff
@@ -31,6 +31,7 @@
           </template>
           <div :style="{width:200+'px'}">
             <b-dropdown-item to="/queue">The Q</b-dropdown-item>
+            <b-dropdown-item to="/booking" v-if="isGAorCSR">Room Booking</b-dropdown-item>
```

Comment:
Can you feature flag this with the `v-if="showExams"` as well, so it doesn't render for offices that aren't enabled?

Reply:
Same comment goes for the actual page itself. Don't render and add a `Coming Soon!` or something to the `/bookings` and the buttons elements